packages/spaces/src/Spaces.js

```diff
@@ -44,6 +44,8 @@ export const getAllSpaces = async (
 
 export const SpacesContext = createContext();
 
+export const useSpacesContext = () => useContext(SpacesContext) || {};
```

Comment:
I'm not sure I agree with defaulting the method to return `{}`. If a user messed up implementing this, they would probably assume the context worked but didn't get any values back, rather than just returning `undefined` when the `SpacesProvider` doesn't exist.

Reply:
agreed. refactored: ff148cb7aa18233230598fdf3869de0e0c486fda